requirements/runtime.txt

```diff
@@ -5,3 +5,4 @@ numpy
 onnx>=1.8.0
 six
 terminaltables
+tqdm
```

Comment:
`protobuf v3+` should be installed. Otherwise, mmdeploy will crash when converting pytorch to onnx if `protobuf v4+` is installed. 
P.S., it looks #532 CI failed because of it.